dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ConfigCenterConfig.java

```diff
@@ -71,6 +81,40 @@ private ConfigCenterConfig() {
 
   public static void setConcurrentCompositeConfiguration(ConcurrentCompositeConfiguration config) {
     finalConfig = config;
+    readFileSourceConfig();
+  }
+
+  @SuppressWarnings("unchecked")
+  public List<String> getFileSource() {
+    return (List<String>) finalConfig.getProperty(FILE_SOURCE);
+  }
+
+  private static void readFileSourceConfig() {
```

Comment:
This PR is not correct. We are not going to support config yaml files. But config yaml contents in config center. Using your test case for example. 

mytest.yaml is a key in config center, and the value of mytest.yaml is the file content.

Reply:
roger that